.github/workflows/frontend.yml

```diff
@@ -40,3 +42,22 @@ jobs:
           yarn test
         env:
           CI: true
+
+  deploy-storybook:
+    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: build storybook'
+        run: |
+          yarn build-storybook
+      - name: deploy gh-pages
+        uses: JamesIves/github-pages-deploy-action@3.4.2
+        with:
+          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
Please switch this to GITHUB_TOKEN. I've created a gh-pages branch, which should work around the issue with publishing in public repositories.